Components/GameBoard/Card.jsx

```diff
@@ -51,7 +51,9 @@ class InnerCard extends React.Component {
             ear: 'E',
             venom: 'M',
             kiss: 'K',
-            bell: 'L'
+            bell: 'L',
+            blood: 'B',
```

Comment:
There's a letter conflict here between blood and betrayal tokens. While right now that's not problematic because they go on different cards, and they use different colors, you never know. I suggest using 'D' instead for blood tokens.

Reply:
Updated.

There was a similar problem with shadow tokens (S for strength, S for shadow). Like above, there's no situation where a card should have the strength psuedo-token and a shadow token. But this should avoid any future conflicts.

As a side note, I would love to rework the UI to get rid of these initials in favor of something more visual. The initials are getting increasingly esoteric to avoid conflicts. 😄